modules/ef-wifi/android/src/main/java/expo/modules/efwifi/ExpoEfWifiModule.kt

```
@@ -0,0 +1,141 @@
+  ): Int {
+    val context = appContext.reactContext?.applicationContext ?: return 1
+    val wifiManager = context.getSystemService(Context.WIFI_SERVICE) as? WifiManager ?: return 1
+
+    val caCerts = try {
+      context.resources.openRawResource(R.raw.cacert).use { stream ->
+        CertificateFactory.getInstance("X.509")
+          .generateCertificates(stream)
+          .map { it as X509Certificate }
+          .toTypedArray()
+      }
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)